packages/cli/test/integration/owner.endpoints.test.ts

```diff
@@ -25,7 +25,13 @@ beforeAll(async () => {
 });
 
 beforeEach(async () => {
+	jest.isolateModules(() => {
+		jest.mock('../../config');
+	});
+
 	await testDb.createOwnerShell();
+
+	config.set('userManagement.isInstanceOwnerSetUp', false);
```

Comment:
This is shared state. `initTestServer()` defaults this to `false`, but state persists among tests, forcing us to surgically reset this one property here despite being the default. 

Combining `jest.mock('../../config')` with `jest.resetAllMocks()` or with `jest.clearAllMocks()` or with `jest.isolateModules()` does not reset the mock state. 

Perhaps we could extract out the convict schema object (as done in the typings inferral PR) and then [`config.load(schemaObj)`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5d9148a2063e37382ef7e1672a47d1094afd70a1/types/convict/index.d.ts#L160-L163) at `beforeEach()`. Clearly out of scope, but please bear in mind since this happens in other tests as well.